models.py

```diff
@@ -30,21 +34,44 @@ def from_raw(cls, raw):
 
     @classmethod
     def get(cls, _id):
-        """Get instance corresponding to ID.
+        """Get document corresponding to ID.
 
         Args:
             _id: primary key/ID of object to be retrieved
         """
         doc = cloudant_ext.db[_id]
         return cls.from_raw(doc)
 
+    @classmethod
+    def contains(cls, _id):
+        """Return True if document with _id exists in DB.
+
+        Args:
+            _id: primary key/ID of object to be retrieved
+        """
+        return cloudant.document.Document(cloudant_ext.db, _id).exists()
+
+    def exists(self):
+        """Return True if this document exists in database."""
+        return User.contains(self._id)
```

Comment:
Should this use the metadata class a la `Document.contains(self._id)`?

With that said, since _id is unique to all documents I'm sure this User.contains will return the same result.

I see a couple other `user` variable name leftovers from stuff it looks like you were abstracting out. Not causing any issues, we can clean those up later for clarity.

Reply:
Good catch.  Submitting a separate PR now.

This worked because `User` didn't implement that method, so really it was the same thing.  It could have broken down the road.  And the other references were just variable names, but sloppy nonetheless.